### PR TITLE
Add subjects.

### LINKED ISSUE #30: Allow posts to appear in subjects.

### PR DESCRIPTION
**Pull request by kevincox**
I also had to fiddle with the fixtures a bit.

Fixes #30

### CODE REVIEW

**Comment by ZachDFK** [comment]
Looks good to me :+1:

**Comment by kevincox** [comment]
I updated this to also include the view. Please re-review.

**Comment by GoodOlChuck** [comment]
looks good :+1:

File: db/schema.rb [hunk 1]
```diff
@@ -36,6 +37,20 @@
...
+  end
+
+  create_table "subjects", force: :cascade do |t|
+    t.string   "name",       null: false
+    t.text     "desc",       null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  add_index "subjects", ["name"], name: "index_subjects_on_name", using: :btree
```
Comment (ZachDFK) [inline db/schema.rb:52]: is that how the routes will go based on the subject names rather than id?

Comment (kevincox) [inline db/schema.rb:52]: No, this is purely a performance optimization.

**Pull request merged**